test/revolution_test.rb

```diff
@@ -212,19 +212,61 @@ def raw_packet_input
         end.to do
             have_one_new_sample(task.data_out_port).matching do |s|
                 json = JSON.parse(s.data.to_byte_array[8..-1])
-                json["method"] == "SET"
+                next if json["method"] != "SET"
+
+                json["payload"]["devices"]["57B974C0A269"]["drive"]["thrust"]
```

Comment:
It should have a sample that has this structure in the hash. This is needed because since we are enabling/disabling auto stabilization depending on the control command, there would samples related to it, and not to the thrust which is what I want to test.